matRad_generateStf.m

```diff
@@ -436,7 +436,20 @@
         axis([-300 300 -300 300 -300 300]);
         %pause(1);
     end
+    
+    % include rangeshifter data if not yet available 
+    if strcmp(pln.radiationMode, 'protons') || strcmp(pln.radiationMode, 'carbon')
+        for j = 1:stf(i).numOfRays
+            if ~isfield(stf(i).ray(j), 'rangeShifter') || isempty(stf(i).ray(j).rangeShifter)
```

Comment:
how can this statement ever be not true? we are creating a new stf from scratch in this function...